recipes/kosh/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "kosh" %}
+{% set version = "1.1" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/LLNL/kosh/archive/v{{ version }}.tar.gz
+    sha256: 96a82d9ed402da865741a85fe90eb021bd911864d3d85fe3e3ab273a443868fb
+
+build:
+  skip: True  # [win]
+  number: 0
+  # noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - kosh = kosh.cli.driver:main
+
+requirements:
+  host:
+    - pip
+    - python
+    - llnl-sina
+  run:
+    - python
+    - llnl-sina
+    - networkx
+    - numpy
+
+test:
+  imports:
+    - kosh
+  commands:
+    - pip check
```

Can the included commands be tested with something like?
```suggestion
    - pip check
    - kosh --help
    - sbang --help
    - init_sina.py --help
```